src/mixins/controls.js

```diff
@@ -1,5 +1,9 @@
 import * as THREE from "three";
+import {CSS2DRenderer} from "three-css2drender"
 import {UtilsMixin} from "./utils";
+import {addAxisLabels} from "./labels";
+import { materialsToThreeDSceneData } from "../utils";
```

Comment:
We should establish the coding style according to https://github.com/Exabyte-io/webstorm-settings